src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -182,6 +189,26 @@ def validate_step(self, step: str) -> bool:
                 return run_new_state is not None and run_new_state.result_state == RunResultState.SUCCESS
         return False
 
+    def _relay_logs(self, workflow, run_id):
+        for log in self._fetch_logs(workflow, run_id):
+            task_logger = logging.getLogger(log.component)
+            # we're relaying almost raw log events, so we need to call the logger directly
+            raw_log_event = task_logger._log  # pylint: disable=protected-access
+            raw_log_event(logging.getLevelName(log.level), log.message, ())
+
+    def _fetch_logs(self, workflow: str, run_id: str) -> Iterator[PartialLogRecord]:
+        log_path = f'{self._install_state.install_folder()}/logs/{workflow}/run-{run_id}'
+        for object_info in self._ws.workspace.list(log_path):
+            if not object_info.path:
+                continue
+            if '.log' not in object_info.path:
+                continue
+            task_name = os.path.basename(object_info.path).split('.')[0]
+            with self._ws.workspace.download(object_info.path) as raw_file:
+                text_io = StringIO(raw_file.read().decode())
+            for record in parse_logs(text_io):
+                yield replace(record, component=f'{record.component}:{task_name}')
```

Comment:
Why postfix the component with a `:{task_name}`?

Reply:
Because assessment has 20 tasks and prefix is swallowed

Reply:
If I understand it correctly, the goal is show the task name again when logging the log records in this module. It would be more clear to me, when we remove the replace here and initiate the logger as `logging.getLogger(f"{log.component":{log.task_name}")`

Reply:
https://github.com/databrickslabs/ucx/issues/1359